### PR TITLE
Add CII badge

### PR DESCRIPTION
**Pull request by apetro**
+ Add CII badge (we're not yet "passing", maybe 80% of the way there.)
+ Re-organize badges

It's a lot of badges. I think this helps make them slightly more digestable.

----

PR considerations checklist:

- [x] Updates `CHANGELOG.md` to reflect this PR's change.
- [x] This Contribution is under the terms of Individual [Contributor License Agreements][] (and also Corporate Contributor License Agreements to the extent applicable) appearing in the [Apereo CLA roster][].

[Apereo CLA roster]: http://licensing.apereo.org/completed-clas
[Contributor License Agreements]: https://www.apereo.org/licensing/agreements

### CODE REVIEW

**Comment by ChristianMurphy** [comment]
@vertein The pitch is:

> The stakes have never been higher for open-source software security. With millions of people around the world relying on open source software — and vulnerabilities like Heartbleed putting everyone at risk — it's time to change the way we support, protect, and fortify open software.

CII badges is a set of best practices around how to manage change control, security, quality, and analysis.
It is a way for project to understand and self report progress toward supporting practices.

**Comment by ChristianMurphy** [comment]
@vertein the pitch is:

> The stakes have never been higher for open-source software security. With millions of people around the world relying on open source software — and vulnerabilities like Heartbleed putting everyone at risk — it's time to change the way we support, protect, and fortify open software.

[source](https://www.coreinfrastructure.org/)


The badge program, is a way for project to self report process towards best practices around change management, security, static analysis, etc.
With the goal being that organizations interested in a project get a better understanding of what processes are in place.

**Comment by apetro** [comment]
Rebased on `master` to pick up that sweet lint-passing goodness.

**Comment by apetro** [comment]
`continuous-integration/appveyor/branch` failure on https://github.com/uPortal-Project/uportal-app-framework/pull/711/commits/1c514c6dc1494d06f6a051156aa9bae96a7f0c5c looks like a false negative.

File: README.md [hunk 1]
```diff
@@ -2,20 +2,29 @@
...
 Questions? [Get in touch][uportal-user@].
 
-<!-- current project status -->
+<!-- coordinates -->
 [![npm version](https://badge.fury.io/js/%40uportal%2Fapp-framework.svg)](https://badge.fury.io/js/%40uportal%2Fapp-framework)
 [![Maven Central](https://maven-badges.herokuapp.com/maven-central/org.apereo.uportal/uportal-app-framework/badge.svg)](https://maven-badges.herokuapp.com/maven-central/org.apereo.uportal/uportal-app-framework)
+<!-- build status -->
 [![Build Status](https://travis-ci.org/uPortal-Project/uportal-app-framework.svg)](https://travis-ci.org/uPortal-Project/uportal-app-framework)
-[![Coverage Status](https://coveralls.io/repos/uPortal-Project/uportal-app-framework/badge.svg?branch=master&service=github)](https://coveralls.io/github/uPortal-Project/uportal-app-framework?branch=master)
+<!-- dependencies statuss -->
```
Comment (ChristianMurphy) [inline README.md]: "status" is uncountable, i.e. it is both singular and plural.
https://www.macmillandictionary.com/dictionary/american/status def 3

Comment (apetro) [reply README.md]: "stati" is definitely the way to go.

**Comment by apetro** [comment]
I agree with @ChristianMurphy 's articulation of freestanding value in Core Infrastructure Initiative badge. However, there's more to it: It's especially interesting in the context of incubation and [of Apereo's (draft) goal of more framework for transparency into health of Apereo projects and products](https://groups.google.com/a/apereo.org/d/msg/open/PWWF0xtz480/GwWd3sE5AgAJ). I suggest that Apereo has insufficient capacity to invent and maintain its own additional frameworks for transparency into health, so the responsible way to pursue that organizational goal, if it's to be pursued, is to piggyback on a framework developed and maintained by another organization. So here's a possibility.

File: README.md (hunk 1 above)
Comment (apetro) [reply README.md]: Apparently ["statuses" is a thing per Merriam-Webster](https://www.merriam-webster.com/dictionary/status).

**Pull request merged**